atdpy/src/lib/Codegen.ml

```diff
@@ -1122,11 +1119,24 @@ let sum env ~class_decorators loc name cases =
   ]
   |> double_spaced
 
+let uses_dataclass_decorator =
+  let rex = Re.Pcre.regexp "\\A[ \t\r\n]*dataclass(\\(|[ \t\r\n]|\\z)" in
```

Comment:
```suggestion
  let rex = Re.Pcre.regexp {|\A[ \t\r\n]*dataclass(\(|[ \t\r\n]|\z)|} in
```

just a style suggestion, when working with regexp I like to avoid the double escaping that is necessary with basic strings

Reply:
great. I don't have this reflex yet.